DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/Digit.h

```diff
@@ -86,22 +86,25 @@ struct Triggers {
 };
 
 struct Digit {
-  o2::dataformats::RangeRefComp<5> ref;
+  o2::dataformats::RangeRefComp<8> ref;
 
   Triggers mTriggers;               // pattern of triggers  in this BC
   o2::InteractionRecord mIntRecord; // Interaction record (orbit, bc)
-
+  int mEventID;
   Digit() = default;
-  Digit(int first, int ne, o2::InteractionRecord iRec, Triggers chTrig)
+  Digit(int first, int ne, o2::InteractionRecord iRec, Triggers chTrig, int event)
   {
     ref.setFirstEntry(first);
     ref.setEntries(ne);
     mIntRecord = iRec;
     mTriggers = chTrig;
+    mEventID = event;
+    LOG(INFO) << " Digits 1st " << ref.getFirstEntry() << " nent " << ref.getEntries();
```

Comment:
Do you need this printout in the constructor?

Reply:
Done